doc/source/examples/incompressible-flow/2d-sudden-expansion-flow/2d-sudden-expansion-flow.rst

```diff
@@ -0,0 +1,332 @@
+      set max refinement level        = 2
+      set max number elements         = 250000
+      set mesh refinement controller  = true
+    end
+
+FEM
+~~~
+
+A linear interpolation order is chosen for the velocity and pressure fields for both :math:`\textrm{Re}` values:
+
+.. code-block:: text
+
+    subsection FEM
+      set pressure order = 1
+      set velocity order = 1
+    end
+
+Boundary Conditions
+~~~~~~~~~~~~~~~~~~~
+
+The inlet velocity is prescribed on boundary 0 as :math:`u = [1, 0, 0]`, and boundary 1 has a prescribed pressure of :math:`p=0` -- identified as ``outlet`` in Lethe. The upper and lower walls (ID 2) have a no-slip Dirichlet boundary condition.
+
+.. code-block:: text
+
+    subsection boundary conditions
+      set number = 3
+      subsection bc 0
+        set id   = 0
+        set type = function
+        subsection u
+          set Function expression = 1
+        end
+        subsection v
+          set Function expression = 0
+        end
+        subsection w
+          set Function expression = 0
+        end
+      end
+      subsection bc 1
+        set id   = 1
+        set type = outlet
+      end
+      subsection bc 2
+        set id   = 2
+        set type = noslip
+      end
+    end
+
+Non-linear Solver
+~~~~~~~~~~~~~~~~~
+
+The default ``newton`` non-linear solver with ``max iterations = 10`` is herein adopted. 
+
+.. code-block:: text
+
+    subsection non-linear solver
+      subsection fluid dynamics
+        set verbosity = verbose
+        set tolerance = 1e-6
+      end
+    end
+
+Whilst the tolerance is kept at ``1e-6`` for :math:`\textrm{Re} = 70`, it is adjusted for ``1e-4`` for :math:`\textrm{Re} = 610`.
+
+Linear Solver
+~~~~~~~~~~~~~
+
+A GMRES iterative solver with ILU preconditioner is used:
+
+.. code-block:: text
+
+    subsection linear solver
+      subsection fluid dynamics
+        set verbosity                             = verbose
+        set method                                = gmres
+        set max iters                             = 500
+        set max krylov vectors                    = 500
+        set relative residual                     = 1e-3
+        set minimum residual                      = 1e-7
+        set preconditioner                        = amg
+        set amg preconditioner ilu fill           = 1
+      end
+    end
+
+The only parameter changed between the low- and high-Reynolds number simulations is the minimum residual, with is changed to ``1e-6`` for :math:`\textrm{Re = 610}`.
+
+-----------------------
+Running the Simulations
+-----------------------
+
+Assuming that the ``lethe-fluid`` executable is within your path, the simulation can be launched by typing
+
+.. code-block:: text
+  :class: copy-button
+
+  mpirun -np j lethe-fluid Reynolds70-jet.prm
+
+where ``j`` is the number of processes for parallel computation. For the case where :math:`\textrm{Re} = 610`, the parameter file should be named ``Reynolds610-jet.prm`` instead.
```

Comment:
Sometimes we like to give a note here that gives an estimate of the simulation runtime.